website/src/roadmap/portability.md

```diff
@@ -165,7 +165,3 @@ Please see the [Spark Runner page]({{ site.baseurl }}/documentation/runners/spar
 how to run portable pipelines on top of Spark.
 
 Python streaming mode is not yet supported on Spark.
-
-## SDK Harness Configuration {#sdk-harness-config}
```

Comment:
Can we leave this?
a) Links to this header could be left floating around out there in my emails etc.
b) I also think it's useful to link these two pages.

Reply:
Yes makes sense, will rollback that change.